src/sdk/namespace_fs.js

```diff
@@ -82,6 +82,7 @@ function make_named_dirent(name) {
  */
 const dir_cache = new LRUCache({
     name: 'nsfs-dir-cache',
+    make_key: ({ dir_path, fs_account_config }) => dir_path + fs_account_config.uid + fs_account_config.gid,
```

Comment:
I think we don't want the uid and gid in the cache key.
Just the path.

Reply:
@guymguym throwing this to the mix:
https://issues.redhat.com/browse/MCGI-48
Not sure how bad was the validation issue @romayalon was seeing. I guess that with uid gid -  validation won't fail as much -  But It can create a lot of cache keys for the same directory.

Reply:
ok after a call with @romayalon and follwing the discussion you had about MCGI-48 removing the uid and gid